src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -1295,6 +1295,22 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 GetEmitter()->emitIns_R_R_R_I(ins, emitSize, targetReg, op1Reg, op2Reg, 0, opt);
                 break;
 
+            case NI_Sve_Count16BitElements:
+            case NI_Sve_Count32BitElements:
+            case NI_Sve_Count64BitElements:
+            case NI_Sve_Count8BitElements:
+            {
+                // Instruction has an additional immediate to multiply the result by. Use 1.
```

Comment:
what about other values `2~16`?

Reply:
That won't happen until we add lowering support (as mentioned above)